### PR TITLE
Leaflet Markercluster

### PR DESCRIPTION
**Pull request by canufeel**
Added Markercluster component with meaningful test.

### CODE REVIEW

**Comment by miguelcobain** [comment]
@canufeel Awesome work!

Unfortunately I think that the way to go is to create a separate addon for this.
I've already created some: http://www.ember-leaflet.com/addons

The idea is for users to be able to install the extra functionality they want.
Since marker cluster isn't in core Leaflet, I feel it is a good candidate for an addon.

Can you do that?

File: addon/components/marker-cluster-layer.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import BaseLayer from 'ember-leaflet/components/base-layer';
+import ContainerMixin from 'ember-leaflet/mixins/container';
+
+/* global L */
+
+export default BaseLayer.extend(ContainerMixin,{
+	L,
```
Comment (miguelcobain) [inline addon/components/marker-cluster-layer.js]: Don't need to include `L` because that is already done at `BaseLayer`.

**Comment by miguelcobain** [comment]
@canufeel Any news?

**Comment by canufeel** [comment]
@miguelcobain Hi! I believe separate addon would be the proper way. However it might take some time for refactoring. I would notify when done.

**Comment by sandersoeteman** [comment]
I am very interested in this addon. How is the refactoring going @canufeel?

**Comment by miguelcobain** [comment]
@sandersoeteman With the code in this PR and the existing ember-leaflet addons, it really shouldn't be too hard to build this addon.
I can provide some guidance at ember's slack if needed.

**Comment by canufeel** [comment]
Hello. you can install it now via `ember install ember-leaflet-marker-cluster` the repo address is https://github.com/canufeel/ember-leaflet-marker-cluster

**Comment by miguelcobain** [comment]
Added a link at http://www.ember-leaflet.com/addons

Thanks a lot, @canufeel.

**Pull request closed without merging**